packages/modules/src/networkAnalysisDataVis/components/settings/popover/SettingMenuPopover.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import Popover from '@material-ui/core/Popover'
+import ListSubheader from '@material-ui/core/ListSubheader'
+import List from '@material-ui/core/List'
+
+import useStyles from './SettingMenuPopover.styles'
+import { SettingItem } from '../../../settingList'
+import PopoverSettingItem from './PopoverSettingItem'
+
+export interface SettingMenuPopoverProps {
+  anchorEl: HTMLButtonElement | null
+  isVisible: boolean
+  handleClose: () => void
+  settingList: SettingItem[]
+  setSettingList: React.Dispatch<React.SetStateAction<SettingItem[]>>
+}
+
+const SettingMenuPopover = ({
+  anchorEl,
+  isVisible,
+  handleClose,
+  settingList,
+  setSettingList
+}: SettingMenuPopoverProps): JSX.Element => {
+  const classes = useStyles()
+
+  return (
+    <Popover
+      className={classes.popover}
+      id={isVisible ? 'simple-popover' : undefined}
+      open={isVisible}
+      anchorEl={anchorEl}
+      onClose={handleClose}
+      anchorOrigin={{
+        vertical: 'top',
+        horizontal: 'left'
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'left'
+      }}
+      // PaperProps={{
+      //   padding
+      // }}
+    >
+      <List
+        component="nav"
+        aria-labelledby="nested-list-subheader"
```

Comment:
I usually generate a unique id in the component to avoid ID clashes if the component is used more than once in a page. Something like 

```
const id = `nested-list-subheader-${guid()}`
```

It's unlikely this particular component will be reused, but still a good habit.

Reply:
That's definitely a good point! This particular ID is unnecessary. I sometimes forget to trim the garbage attributes that come from the MUI documentation.